domains/runtime/core-eth-relay/src/runtime.rs

```diff
@@ -290,6 +293,60 @@ impl pallet_sudo::Config for Runtime {
     type RuntimeCall = RuntimeCall;
 }
 
+/// Dummy time provider that always returns zero.
+pub struct DummyTimeProvider;
+impl UnixTime for DummyTimeProvider {
+    fn now() -> Duration {
+        Duration::ZERO
+    }
+}
+
+// Ethereum mainnet configuration
+parameter_types! {
+    pub const MaxSyncCommitteeSize: u32 = 512;
+    pub const MaxProofBranchSize: u32 = 20;
+    pub const MaxExtraDataSize: u32 = 32;
+    pub const MaxLogsBloomSize: u32 = 256;
+    pub const MaxFeeRecipientSize: u32 = 20;
+    pub const MaxPublicKeySize: u32 = 48;
+    pub const MaxSignatureSize: u32 = 96;
+    pub const MaxSlotsPerHistoricalRoot: u64 = 8192;
+    pub const MaxFinalizedHeaderSlotArray: u32 = 1000;
+    pub const WeakSubjectivityPeriodSeconds: u32 = 97200;
+    pub const ChainForkVersions: ForkVersions = ForkVersions{
+        genesis: Fork {
+            version: [0, 0, 16, 32], // 0x00001020
+            epoch: 0,
+        },
+        altair: Fork {
+            version: [1, 0, 16, 32], // 0x01001020
+            epoch: 36660,
+        },
+        bellatrix: Fork {
+            version: [2, 0, 16, 32], // 0x02001020
+            epoch: 112260,
+        },
+    };
+}
+
+impl pallet_ethereum_beacon_client::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    // TODO: Replace this with pallet_timestamp once core domains support inherent extrinsics
```

Comment:
I'm not sure they ever will. Execution is deterministic, timestamp must come from the consensus chain block then.